doc/man3/ASN1_STRING_length.pod

```diff
@@ -77,13 +77,33 @@ character in big endian format, UTF8String will be in UTF8 format.
 Similar care should be take to ensure the data is in the correct format
 when calling ASN1_STRING_set().
 
+=head1 RETURN VALUES
+
+ASN1_STRING_length() returns the length of the content of B<x>.
```

Comment:
On *side note* one can wonder if it's appropriate to elaborate on length units. I mean that it's buffer length, not character count...